broker/service_instance_client_test.go

```diff
@@ -2,21 +2,23 @@ package broker_test
 
 import (
 	"errors"
+	"log"
+
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	brokerfakes "github.com/pivotal-cf/on-demand-service-broker/broker/fakes"
-	"log"
 )
 
 var _ = Describe("ServiceInstanceClient", func() {
 	var (
-		instanceID     string
-		instanceName   string
-		spaceGUID      string
-		fakeUAAClient  *brokerfakes.FakeUAAClient
-		expectedClient map[string]string
-		rawContext     map[string]interface{}
-		logger         *log.Logger
+		instanceID         string
+		instanceName       string
+		spaceGUID          string
+		customClientSecret string
```

Comment:
it might just be the formatting here on the github UI, but are we ever assigning anything to "customClientSecret" ?

all the other test variables / inputs around this, like "instanceID", are assigned to dummy values in the BeforeEach - it would probably be good to stay consistent

Reply:
Hi @ryanwittrup thanks for poiting that out. This is some cruft from an earlier experiemnt where we were thinking of passing the client_secret at provision time. This is no longer needed for the allowpublic flow.